### PR TITLE
Update readme

### LINKED ISSUE #36: Add README for squid-blockexplorer
Add documentation/README to work with `squid-blockexplorer`

### PR DESCRIPTION
**Pull request by isSerge**
Add readme for the whole repo and squid (frontend has default `create-react-app` readme)

Closes #36

### CODE REVIEW

File: squid-blockexplorer/README.md [hunk 1]
```diff
@@ -1 +1,37 @@
...
+# 4. Start the processor
+make process
+```
+
+Make sure you provide following environmental variables:
+- `CHAIN_RPC_ENDPOINT` - Public node RPC endpoint
+- `ARCHIVE_ENDPOINT` - GraphQL endpoint to a local or remote archive
+> Please, refer to Subsquid [docs](https://docs.subsquid.io/archives/) to learn, more about Squid Archives
+
+In order to use GraphiQL explorer, run following in a separate terminal:
```
Comment (jfrank-summit) [inline squid-blockexplorer/README.md:26]:
```suggestion
In order to use GraphQL explorer, run following in a separate terminal:
```

Comment (isSerge) [reply squid-blockexplorer/README.md:26]: This is not a typo, I meant this:
image

Comment (jfrank-summit) [reply squid-blockexplorer/README.md:26]: My bad lol

**Review by jfrank-summit — APPROVED** [review]
Lgtm

**Pull request merged**